kolibri/plugins/coach/assets/src/views/common/BackLink.vue

```diff
@@ -2,33 +2,16 @@
 
   <!-- TODO: move this to be a core KBackLink -->
   <span v-show="!$isPrint" class="offset">
-    <UiIconButton
-      type="flat"
-      class="icon"
-      tabIndex="-1"
-      aria-hidden="true"
-      @click="go"
-    >
-      <KIcon
-        icon="back"
-        :style="{ fill: $themeTokens.primary, top: 0, height: '24px', width: '24px' }"
-      />
-    </UiIconButton>
-    <KRouterLink :to="to" :text="text" />
+    <KRouterLink :to="to" icon="back" :text="text" />
```

Comment:
Looks like we might be able to delete the `BackLink` component entirely and replace every instance with just

```html
<KRouterLink :to="target" icon="back" :text="text" />
```